### DISCUSSION TITLE
糖果VR资源网 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#123] [模板讨论]
糖果VR资源网 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by Nighting0518** [comment]
2022-11-22后签到一直反馈unterminated character set at position 16

**Reply by wjf0214** [reply]
目前我这一直正常。没有问题。

**Reply by Nighting0518** [reply]
抱歉，我之前只是删除重建了任务，这次模板重新订阅以后正常了 可能是之前不小心修改了模板

**Reply by wjf0214** [reply]
OK，没问题就行。

**Comment by miaoxiaozhao** [comment]
提示nterminated character set at position 16，不过签到实际成功了。

**Reply by wjf0214** [reply]
没有遇到过，大概率可能是网站自己本身的问题。请求的时候返回失败了，但是实际成功了。
